cmd/fleetctl/mdm_test.go

```diff
@@ -730,6 +746,21 @@ func TestMDMUnlockCommand(t *testing.T) {
 		return h.MDMInfo, nil
 	}
 
+	ds.GetHostOrbitInfoFunc = func(ctx context.Context, hostID uint) (*fleet.HostOrbitInfo, error) {
+		hostIDMod := hostID % 3
+		switch hostIDMod {
+		case 0:
+			return nil, &notFoundError{}
+		case 1:
+			return &fleet.HostOrbitInfo{}, nil
+		case 2:
+			return &fleet.HostOrbitInfo{ScriptsEnabled: ptr.Bool(true)}, nil
```

Comment:
Don't we need a test with `ScriptsEnabled: ptr.Bool(false)`?

Reply:
The integration test covers that.